### PR TITLE
Fix/go

### PR DESCRIPTION
**Pull request by craxkumar**
Resolved service discovery issue

### CODE REVIEW

**Review by Sri-Harsha06 — APPROVED** [review]

File: generator-jhipster-go/generators/server/templates/go/go/customlogger/customlogger.go [hunk 1]
```diff
@@ -36,17 +36,17 @@ func createLogger(fname string) *loggers {
...
+    // logger.InfoLogger.Println(msg)
+    // }
+
+    // if(level=="warn"){
+    //  logger.WarningLogger.Println(msg)
+    // }
+
+    // if(level=="error"){
+    // logger.ErrorLogger.Println(msg)
+    // }
```
Comment (rohithteja-cmi) [inline generator-jhipster-go/generators/server/templates/go/go/customlogger/customlogger.go:51]: why is this commented? we must log into file as well.

File: generator-jhipster-go/generators/server/templates/go/go/helper/HttpHelper.go [hunk 2]
```diff
@@ -89,6 +89,6 @@ func MakePutCall(urlToPost string, body OperationBody, headers map[string]string
 		customlogger.Printfun("error","Error while making PUT call " + err.Error())    
 		return errors.New("Error while making PUT call " + err.Error()), nil
 	}
-	// customlogger.Printfun("info","Error while making PUT call " + "Successfull POST call with HTTP Status : " + resp.Status)    
+	customlogger.Printfun("info","Error while making PUT call " + "Successfull POST call with HTTP Status : " + resp.Status)
```
Comment (rohithteja-cmi) [inline generator-jhipster-go/generators/server/templates/go/go/helper/HttpHelper.go]: Successfull -> Successful
In the 2nd argument, `"Error while making PUT call " + "Successfull POST call with HTTP Status : "`, first concatenation is not required right, they can be put in one string -> `"Error while making PUT call Successful POST call with HTTP Status : "`

**Review by rohithteja-cmi — DISMISSED** [review]

File: generator-jhipster-go/generators/server/templates/go/go/customlogger/customlogger.go (hunk 1 above)
Comment (craxkumar) [reply generator-jhipster-go/generators/server/templates/go/go/customlogger/customlogger.go:51]: we are reading container logs i believe, so commented them so of now!
will check and revert back!

File: generator-jhipster-go/generators/server/templates/go/go/helper/HttpHelper.go (hunk 2 above)
Comment (craxkumar) [reply generator-jhipster-go/generators/server/templates/go/go/helper/HttpHelper.go]: sure!

**Pull request merged**